lib/traject/config/sirsi_config.rb

```diff
@@ -246,7 +246,7 @@ def extract_sortable_title(fields, record)
 # # Author Sort Field
 to_field 'author_sort' do |record, accumulator|
   accumulator << extract_sortable_author('100abcdefgjklnpqtu:110abcdefgklnptu:111acdefgjklnpqtu',
-                                         '240adfghklmnoprs:245abcfghknps',
+                                         '240acdfghklmnoprs:245abfghknps',
```

Comment:
240 $c is not a valid subfield. I found we have 6 records with 240 $c, but I think they are errors. I will ask Data Control regarding those records.

Reply:
Data Control confirmed that the records with 240 $c were mistakes. But as you can see, it's possible for records to have a 240 $c. I think we should exclude it from the author_sort.

Reply:
The solrmarc code currently includes them; what do you think about making a ticket to remove this behavior after we can verify we produce identical data?

Reply:
Ok. #110